packages/react-aria-components/src/Tree.tsx

```diff
@@ -397,6 +567,14 @@ export const TreeItem = /*#__PURE__*/ createBranchComponent('item', <T extends o
   // eslint-disable-next-line
   }, []);
 
+  let dragButtonRef = useRef<HTMLButtonElement>(null);
+  useEffect(() => {
+    if (dragState && !dragButtonRef.current && process.env.NODE_ENV !== 'production') {
+      console.warn('Draggable items in a Table must contain a <Button slot="drag"> element so that keyboard and screen reader users can drag them.');
```

Comment:
```suggestion
      console.warn('Draggable items in a Tree must contain a <Button slot="drag"> element so that keyboard and screen reader users can drag them.');
```